File: DigitSum/DigitSum.js
//  Find Sum of Digit:

// n = parseInt(prompt("Enter number: "))
n = 10
temp = n
sum = 0
while (n > 0) {
    sum += n % 10
    n = Math.floor(n / 10)
}
console.log(`The sum of ${temp} = ${sum}`)

File: DigitSum/DigitSum.cs
//  Find Sum of Digit:
using System;
class Program
{
    static void main()
    {
        int num, i, temp, sum = 0;
        Console.Write("Enter number: ");
        num = int.Parse(Console.ReadLine());
        temp = num;
        while (num > 0)
        {
            sum += num % 10;
            num /= 10;
        }
        Console.WriteLine($"sum of {temp} is {sum}");
    }
}